ts/src/test/base/test.base.ts

```diff
@@ -30,19 +30,21 @@ function testCalculateFee () {
         'quote':  'BAR',
         'taker':   taker,
         'maker':   maker,
-        'precision': {
-            'amount': 8,
-            'price': 8,
-        },
```

Comment:
here `exchange` variable was not yet defined, so we have to remove this from here and add after `exchange`'s initialization